test/system/irohad_test.cpp

```diff
@@ -135,15 +156,69 @@ DROP TABLE IF EXISTS index_by_id_height_asset;
   std::string config_copy_;
 };
 
-/*
+/**
  * @given path to irohad executable and paths to files irohad is needed to be
  * run (config, genesis block, keypair)
  * @when run irohad with all parameters it needs to operate as a full node
  * @then irohad should be started and running until timeout expired
  */
 TEST_F(IrohadTest, RunIrohad) {
-  child c(irohad_executable.string() + setDefaultParams());
-  std::this_thread::sleep_for(timeout);
-  ASSERT_TRUE(c.running());
-  c.terminate();
+  launchIroha();
+}
+
+/**
+ * Test verifies that a transaction can be sent to running iroha and committed
+ * @given running Iroha
+ * @when a client sends a transaction to Iroha
+ * @then the transaction is committed
+ */
+TEST_F(IrohadTest, SendTx) {
+  launchIroha();
+  auto key_manager = iroha::KeysManagerImpl(kAdminId, path_example_);
+  auto key_pair = key_manager.loadKeys();
+  ASSERT_TRUE(key_pair);
+
+  iroha::protocol::TxStatusRequest tx_request;
+  iroha::protocol::ToriiResponse torii_response;
+
+  auto tx =
+      complete(baseTx(kAdminId).setAccountQuorum(kAdminId, 1), key_pair.get());
+  tx_request.set_tx_hash(shared_model::crypto::toBinaryString(tx.hash()));
+
+  auto client = torii::CommandSyncClient(kAddress, kPort);
+  client.Torii(tx.getTransport());
+
+  auto resub_counter(resubscribe_attempts);
+  do {
+    std::this_thread::sleep_for(resubscribe_timeout);
+    client.Status(tx_request, torii_response);
+  } while (torii_response.tx_status() != iroha::protocol::TxStatus::COMMITTED
```

Comment:
`iroha::protocol::TxStatus::COMMITTED` can be moved to a separate variable, so that the literal is not duplicated both in line 195 and 198.